tests/lib/mock_nm.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3 -tt
+# vim: fileencoding=utf8
+
+UP = 1
+DOWN = 0
+
+NM_DEVICE_TYPE_GENERIC = 14
+
+
+class MockNmConnection(object):
+    def get_ip4_config(self):
+        return None
+
+    def get_connection(self):
+        return MockNmConnection()
+
+
+class MockNmDevice(object):
+    def __init__(self, devstate=DOWN, active_connection=MockNmConnection(),
+                 iface="lo"):
```

Comment:
single quotes please, the same for others unless there is a single quote inside the string

Reply:
This is a new rule for me, where does it come from?